drivers/scheduler/oke/oke.go

```diff
@@ -0,0 +1,585 @@
+	} `json:"node-shape-config"`
+	NodeSource struct {
+		ImageID    string `json:"image-id"`
+		SourceName string `json:"source-name"`
+		SourceType string `json:"source-type"`
+	} `json:"node-source"`
+	NodeSourceDetails struct {
+		BootVolumeSizeInGBs interface{} `json:"boot-volume-size-in-gbs"`
+		ImageID             string      `json:"image-id"`
+		SourceType          string      `json:"source-type"`
+	} `json:"node-source-details"`
+	QuantityPerSubnet int         `json:"quantity-per-subnet"`
+	SshPublicKey      string      `json:"ssh-public-key"`
+	SubnetIds         []string    `json:"subnet-ids"`
+	SystemTags        interface{} `json:"system-tags"`
+}
+
+type WorkRequest struct {
+	OpcWorkRequestId string `json:"opc-work-request-id"`
+}
+
+type WorkRequestGet struct {
+	Data struct {
+		Resources []struct {
+			Identifier string `json:"identifier"`
+		} `json:"resources"`
+		Status string `json:"status"`
+	} `json:"data"`
+}
+
+type responseWithListId struct {
+	Data []struct {
+		DisplayName  string `json:"display-name"`
+		Id           string `json:"id"`
+		FreeformTags struct {
+			K8SVersion string `json:"k8s_version"`
+		} `json:"freeform-tags,omitempty"`
+	} `json:"data"`
+}
+
+func (o *oke) String() string {
+	return SchedName
+}
+
+// Init initializes the node driver for oke under the given scheduler
+func (o *oke) Init(schedOpts scheduler.InitOptions) error {
+
+	instanceGroup := os.Getenv("INSTANCE_GROUP")
+	if len(instanceGroup) != 0 {
+		o.instanceGroupName = instanceGroup
+	} else {
+		o.instanceGroupName = "default"
+	}
+	compartmentId := os.Getenv("PX_ORACLE_compartment_id")
+	if len(compartmentId) == 0 {
+		return fmt.Errorf("compartment id not provided as env var [PX_ORACLE_compartment_id]: %s", compartmentId)
+	}
+	o.compartmentId = compartmentId
+
+	ops, err := oracleOps.NewClient()
+	if err != nil {
+		return err
+	}
+	o.ops = ops
+	err = o.K8s.Init(schedOpts)
+	if err != nil {
+		return err
+	}
+
+	err = configureUser()
+	if err != nil {
+		return err
+
+	}
+	err = o.setCluster()
+
+	return err
+}
+
+func configureUser() error {
+	userKey := os.Getenv("PX_ORACLE_user_ocid")
+	if len(userKey) == 0 {
+		return fmt.Errorf("user key not provided as env var [PX_ORACLE_user_oci]: %s", userKey)
+	}
+
+	fingerprint := os.Getenv("PX_ORACLE_fingerprint")
+	if len(fingerprint) == 0 {
+		return fmt.Errorf("fingerprint key not provided as env var [PX_ORACLE_fingerprint]: %s", fingerprint)
+	}
+	privateKey := os.Getenv("PX_ORACLE_private_key_path")
+	if len(privateKey) == 0 {
+		return fmt.Errorf("private key not provided as env var [PX_ORACLE_private_key_path]: %s", privateKey)
+	}
+	region := os.Getenv("PX_ORACLE_cluster_region")
+	if len(region) == 0 {
+		return fmt.Errorf("region not provided as env var [PX_ORACLE_cluster_region]: %s", region)
+	}
+	tenancy := os.Getenv("PX_ORACLE_tenancy")
+	if len(tenancy) == 0 {
+		return fmt.Errorf("tenancy not provided as env var [PX_ORACLE_tenancy]: %s", tenancy)
+	}
```

Comment:
Don't think we need to print things we know are empty since we already doing `if len(key) == 0 {}`

Reply:
fixed